config/cable.yml

```diff
@@ -6,5 +6,5 @@ test:
 
 production:
   adapter: redis
-  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
-  channel_prefix: asana_production
+  url: <%= ENV.fetch("REDIS_URL") { "redis://159.89.96.70:6381" } %>
```

Comment:
check if it works with localhost
because redis is located on the same server as production so it's localhost for it